SettingsList.py

```diff
@@ -1456,33 +1509,59 @@ def __init__(self, name, gui_text, min, max, default, step=1,
-                    '''},
-    'Shadow Temple MQ beyond Invisible Moving Platform without Lens of Truth': {
-        'name'    : 'logic_lens_shadow_mq_back',
+                    in Shadow Temple MQ for most areas in the dungeon.
+                    See "Shadow Temple MQ Invisible Moving Platform
+                    without Lens of Truth", "Shadow Temple MQ Invisible
+                    Blades Silver Rupees without Lens of Truth", and
+                    "Shadow Temple MQ 2nd Dead Hand without Lens of Truth"
+                    for exceptions.
+                    '''},
+    'Shadow Temple MQ Invisible Moving Platform without Lens of Truth': {
+        'name'    : 'logic_lens_shadow_mq_platform',
         'tags'    : ("Lens of Truth","Shadow Temple",),
         'tooltip' : '''\
                     Removes the requirements for the Lens of Truth
-                    in Shadow Temple MQ beyond the invisible moving platform.
+                    in Shadow Temple MQ to cross the invisible moving
+                    platform in the huge pit room in either direction.
+                    '''},
+    'Shadow Temple MQ Invisible Blades Silver Rupees without Lens of Truth': {
+        'name'    : 'logic_lens_shadow_mq_invisible_blades',
+        'tags'    : ("Lens of Truth","Shadow Temple",),
+        'tooltip' : '''\
+                    Removes the requirement for the Lens of Truth or
+                    Nayru's Love in Shadow Temple MQ for the Invisible
+                    Blades room silver rupee collection.
+                    '''},
+    'Shadow Temple MQ 2nd Dead Hand without Lens of Truth': {
+        'name'    : 'logic_lens_shadow_mq_dead_hand',
+        'tags'    : ("Lens of Truth","Shadow Temple",),
+        'tooltip' : '''\
+                    Dead Hand can spawn out of bounds in the back room of
+                    Shadow Temple, making him unreachable until the room
+                    is reloaded. The only way to tell if this happened is
+                    with Lens of Truth. This trick removes that safety net.
                     '''},
-    'Shadow Temple before Invisible Moving Platform without Lens of Truth': {
+    'Shadow Temple Stationary Objects without Lens of Truth': {
         'name'    : 'logic_lens_shadow',
         'tags'    : ("Lens of Truth","Shadow Temple",),
         'tooltip' : '''\
                     Removes the requirements for the Lens of Truth
-                    in Shadow Temple before the invisible moving platform.
+                    in Shadow Temple for most areas in the dungeon
+                    except for crossing the moving platform in the huge
+                    pit room.
                     '''},
-    'Shadow Temple beyond Invisible Moving Platform without Lens of Truth': {
-        'name'    : 'logic_lens_shadow_back',
+    'Shadow Temple Invisible Moving Platform without Lens of Truth': {
+        'name'    : 'logic_lens_shadow_platform',
```

Comment:
I don't know if we have any of those tricks in presets and/or unit test files, but if we do they probably need to be updated

Reply:
The vanilla shadow trick was in a bunch of presets and unit tests. Some of the other renamed tricks were in the hell mode preset. All should be caught now, and unit tests remain valid.